src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -332,7 +355,7 @@ public void validateInternalCollaborators(DataAccessRequest payload) {
    * @return The updated DataAccessRequest
    */
   public DataAccessRequest updateByReferenceId(User user, DataAccessRequest dar) {
-    if (!dar.getDraft()) {
+    if (Boolean.FALSE.equals(dar.getDraft())) {
```

Comment:
This check is different than the other one, in that it maps `null` to `true`, not `false`. If this field can be `null`, it should work consistently. If it can't be `null`, we should make the getter return `boolean` instead of `Boolean`.